go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/ff4f01/shimmer
```

Comment:
This is incorrect. You are sending the PR from the perspective of your repo. :smile: 

You need to clone my repo and make the changes there. And then add a new origin pointing to your forked repo, and push it there. And then send a PR.